autotest/cldrpattern/test.py

```diff
@@ -0,0 +1,90 @@
+import os,json,HTMLTestRunner,unittest
+from excelutility import excelutil
+from ddt import ddt,data
+from datetime import datetime
+
+# Set testdata directory:
+# rootdir = 'D:\\PycharmProjects\\SmokeTesting\\singleton-i18n-patterns-core-0.5.1' #The path of cldr data storage
+# test_dir = 'D:\\PycharmProjects\\SmokeTesting' #The directory of generated test report
+rootdir = './singleton-i18n-patterns-core-0.5.1' #The path of cldr data storage
+test_dir = './' #The directory of generated test report
+
+def curfoldercount(dir):
+    return str(len(os.listdir(dir)))
+
+
+def rootfoldercount(dir):
+    L = []
+    for paths in os.walk(dir):
+        L.append(paths)
+    # L includes the root, should -1 to get the folder count under it
+    return str(len(L)-1)
+
+def rootfilecount(dir):
+    L = []
+    for paths, folders, files in os.walk(dir):
+        L.extend(files)
+    return str(len(L))
+
+def readjson(dir, filename, key):
+    with open(dir+filename, 'r', encoding='utf-8') as f:
+        temp = json.load(f)
+        keylist = key.split('.')
+        for i in keylist:
+            temp = temp[i]
+    return str(temp)
+
+
+
+@ddt
+class Test(unittest.TestCase):
+
+    @data(*excelutil())
+    def test(self, case):
+
+        if (case['casename'] == 'Check the number of folders in cldr folder'):
```

Comment:
can you give a description about what these verification means? so people would know how you test it.

Reply:
Adding case category to identify the case purpose, please see the latest version.